models/models.js

```diff
@@ -104,3 +111,24 @@ exports.fetchCommentsForArticle = async (articleID) => {
 
   return commentResult.rows;
 };
+
+/* ***************POST COMMENTS FOR EACH ARTICLE BY ID****************/
+
+exports.addCommentToArticle = async (articleID, username, body) => {
+  const userValue = [username, username, "somephoto"];
```

Comment:
I don't understand what this user part is doing. We just need to insert the values we're getting though into comments - each comment has an article_id and an author, so we have everything we need.